software/network/src/main/resources/org/apache/brooklyn/entity/network/bind/named.conf

```diff
@@ -20,14 +20,15 @@
 // Generated by Brooklyn on ${.now?string.short}
 //
 options {
-    listen-on port ${entity.dnsPort} { 127.0.0.1; ${driver.address}; };
+    // listen-on default is all interfaces
+    // listen-on port ${entity.dnsPort} { 127.0.0.1; ${driver.address}; ${driver.subnetAddress}; };
```

Comment:
Actually, use the following for both:
```
// This will cause named to listen on all IPv4 and IPv6 configured interfaces.
listen-on-v6 ${entity.dnsPort} { any; };
```